### PR TITLE
449 placeholder persist

### PR DESCRIPTION
**Pull request by j-leidy**
Modified blurring logic again to fix a niche issue I found.
- The user can choose to show all values selected or show the placeholder.

### CODE REVIEW

File: src/components/Dropdown/Dropdown.tsx [hunk 1]
```diff
@@ -628,7 +628,11 @@ const Dropdown = ({
       // when searchable, only blur if the event is from the input
       setFocusTimeoutId(
         window.setTimeout(() => {
-          if (focusWithin && (!searchable || e.target.id === `${name}-search-input`)) {
+          if (
+            focusWithin &&
+            ((!searchable && e.target.id === `${name}-dropdown-button`) ||
+              (searchable && e.target.id === `${name}-search-input`))
```
Comment (aVileBroker) [inline src/components/Dropdown/Dropdown.tsx:634]: this is getting kind of big, could you make a variable above the if that encapsulate the logic here? Like
```ts
const blurredFromButton = (!searchable && e.target.id === `${name}-dropdown-button`);
etc
```

File: src/components/Dropdown/Dropdown.tsx [hunk 2]
```diff
@@ -953,7 +959,8 @@ const Dropdown = ({
           ...(valueContainerProps ? valueContainerProps.containerProps : {}),
         }}
       >
-        {searchCharacterCount === 0 && (!values || !values.length) && !focusWithin && (
+        {((searchCharacterCount === 0 && (!values || !values.length) && !focusWithin) ||
+          (!showSelectedValues && !focusWithin)) && (
```
Comment (aVileBroker) [inline src/components/Dropdown/Dropdown.tsx:963]: again if you can reduce the growing craziness here with some variables that would be awesome

**Review by aVileBroker — CHANGES_REQUESTED** [review]

File: src/components/Dropdown/Dropdown.tsx [hunk 3]
```diff
@@ -678,6 +685,8 @@ const Dropdown = ({
     (clickedId: string | number) => {
       if (!multi) {
         setIsOpen(false);
+        // not sure how i feel about this... the focus currently stays after selecting an item when multi is false
```
[the remark this answers is not in the document]
Comment (j-leidy) [reply src/components/Dropdown/Dropdown.tsx:685]: Old comment I made yesterday. Will remove. I made a truth table for all combinations of multi, searchable and showSelectedValues. Ensured the functionality of blurring was correct.

File: src/components/Dropdown/Dropdown.tsx (hunk 2 above)
Comment (j-leidy) [reply src/components/Dropdown/Dropdown.tsx:963]: Looking into it now!

File: src/components/Dropdown/Dropdown.tsx (hunk 1 above)
Comment (j-leidy) [reply src/components/Dropdown/Dropdown.tsx]: Done 👍🏼

**Review by aVileBroker — APPROVED** [review]

**Pull request merged**